File: Models/Bibliotekar.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Models
{
    public enum RadnoVreme
    {
        Prepodne,
        Popodne
    }

    [Table("Bibliotekari")]
    public class Bibliotekar
    {
        [Key]
        public int Id_Bibliotekar { get; set; }

        [Required]
        [Range(10000,99999)]
        public int BrojKnjizice { get; set; }

        [MaxLength(20)]
        [Required]
        public string Ime { get; set; }

        [MaxLength(20)]
        [Required]
        public string Prezime { get; set; }

        public char Pol { get; set; }

        public RadnoVreme Smena { get; set; }

        [JsonIgnore]
        public List<Iznajmljivanje> IzdateKnjige { get; set; }

    }
}

File: Klijent/Bibliotekar.js
export class Bibliotekar {
    constructor(ime, prezime, brojKnjizice, pol, smena) {
        this.ime = ime;
        this.prezime = prezime;
        this.brojKnjizice = brojKnjizice;
        this.pol = pol;
        this.smena = smena;
    }
    crtaj(host) {

        var tr = document.createElement("tr");
        host.appendChild(tr);

        var el = document.createElement("td");
        el.innerHTML = this.brojKnjizice;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.ime;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.prezime;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.pol;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.smena;
        tr.appendChild(el);
    }

}